tslint.json

```diff
@@ -0,0 +1,72 @@
+{
+  "rules": {
+    "class-name": false,
+    "comment-format": [
+      true
+    ],
+    "curly": false,
+    "eofline": false,
+    "forin": false,
+    "indent": [
+      true,
+      "spaces"
+    ],
+    "label-position": true,
+    "max-line-length": false,
+    "member-access": false,
+    "member-ordering": [
+      true,
+      "static-after-instance",
+      "variables-before-functions"
+    ],
+    "no-arg": true,
+    "no-bitwise": false,
+    "no-console": false,
+    "no-construct": true,
+    "no-debugger": true,
+    "no-duplicate-variable": true,
+    "no-empty": false,
+    "no-eval": true,
+    "no-inferrable-types": true,
+    "no-shadowed-variable": true,
+    "no-string-literal": false,
+    "no-switch-case-fall-through": true,
+    "no-trailing-whitespace": true,
+    "no-unused-expression": true,
+    "no-use-before-declare": false,
+    "no-var-keyword": true,
+    "object-literal-sort-keys": false,
+    "one-line": [
+      true,
+      "check-open-brace",
+      "check-catch",
+      "check-else",
+      "check-whitespace"
+    ],
+    "quotemark": false,
+    "radix": true,
+    "semicolon": [
+      "always"
+    ],
+    "triple-equals": [],
+    "typedef-whitespace": [
+      true,
+      {
+        "call-signature": "nospace",
+        "index-signature": "nospace",
+        "parameter": "nospace",
+        "property-declaration": "nospace",
+        "variable-declaration": "nospace"
+      }
+    ],
+    "variable-name": false,
+    "whitespace": [
+      true,
+      "check-branch",
+      "check-decl",
+      "check-operator",
+      "check-separator",
+      "check-type"
+    ]
+  }
+}
```

Comment:
Where are these rules from? A lot of them look like ones ported from our eslint config which seems unfortunate in terms of now having the same rules defined in two places. Not sure what other options would be. Could we lint our plain js with tslint too?

Reply:
The rules are amalgamated from various places. I'm pretty sure tslint doesn't work for js, but will look into it for another PR.

Reply:
fair enough